ocaml/xapi/message_forwarding.ml

```diff
@@ -3110,6 +3110,12 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       let local_fn = Local.PIF.set_property ~self ~name ~value in
       do_op_on ~local_fn ~__context ~host (fun session_id rpc -> Client.PIF.set_property rpc session_id self name value)
 
+    let set_disallow_unplug ~__context ~self ~value =
+      info "PIF.set_disallow_unplug: PIF: %s; value = %s" (Ref.string_of self) (string_of_bool value);
+      let host = Db.PIF.get_host ~__context ~self in
+      let local_fn = Local.PIF.set_disallow_unplug ~self ~value in
+      do_op_on ~local_fn ~__context ~host (fun session_id rpc -> Client.PIF.set_disallow_unplug rpc session_id self value)
```

Comment:
I think we should not forward it but just do it locally on the master which has the DB, since we only do DB queries. (Like for `vdi_set_cbt_enabled`)

Reply:
Yes, if the function act on the xapi DB only, than we should just call `Local...` directly.